base/src/main/java/org/aya/core/term/MetaLitTerm.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020-2022 Tesla (Yinsen) Zhang.
+// Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+package org.aya.core.term;
+
+import kala.collection.immutable.ImmutableSeq;
+import kala.control.Option;
+import kala.tuple.Tuple2;
+import org.aya.core.def.GenericDef;
+import org.aya.core.repr.AyaShape;
+import org.aya.core.repr.ShapeRecognition;
+import org.aya.util.error.SourcePos;
+import org.jetbrains.annotations.NotNull;
+
+public record MetaLitTerm(
```

Comment:
Is there a "metavar" for metalits?

Reply:
`MetaLit` reuses the `MetaTerm` as its type, so the metavar is just ordinary meta (unlike `MetaPat`)

Reply:
Cooooooooooooooool